services/storage/src/simcore_service_storage/api/rest/_paths.py

```diff
@@ -22,15 +22,16 @@
 
 @router.get(
     "/locations/{location_id}/paths",
-    response_model=CursorPage[PathMetaDataGet],
+    response_model=LargeLimitedPage[PathMetaDataGet],
 )
 async def list_paths(
-    page_params: Annotated[CursorParams, Depends()],
+    page_params: Annotated[LargeLimitedPage.__params_type__, Depends()],
```

Comment:
this one looks pretty strange.

Reply:
yep. but otherwise no openapi generated..